core/lib/instrument_api.h

```diff
@@ -6201,6 +6201,13 @@ DR_API
 bool
 dr_prepopulate_cache(app_pc *tags, size_t tags_count);
 
+DR_API
+/**
+ * Get the number of blocks built so far, globally. The API is not thread-safe.
```

Comment:
"blocks" should be clearer: "basic blocks".  The comment could also explain that this is the total number ever built and includes duplicates if there were deletions for consistency (i.e., app code mod) or capacity reasons or thread-private caches.

Reply:
Done.